k8s/upgrade/src/constant.rs

```diff
@@ -12,17 +14,45 @@ macro_rules! upgrade_labels {
 }
 
 /// Append the release name to k8s objects.
-pub(crate) fn upgrade_name_concat(release_name: &str, component_name: &str) -> String {
-    format!("{release_name}-{component_name}")
+pub(crate) fn upgrade_name_concat(
+    release_name: &str,
+    component_name: &str,
+    upgrade_to_develop: bool,
+) -> String {
+    if upgrade_to_develop {
+        "develop".to_string()
```

Comment:
Should this be taken from a constant?

Reply:
We should be able to have upgrade_to_branch instead of upgrade to devel? Ex: when we want to test the release branch but there's no version.

Reply:
Yep, that makes sense.